tests/components/apsystems/conftest.py

```diff
@@ -14,3 +14,27 @@ def mock_setup_entry() -> Generator[AsyncMock, None, None]:
         return_value=True,
     ) as mock_setup_entry:
         yield mock_setup_entry
+
+
+@pytest.fixture
+def mock_apsystems_timeout():
+    """Override APsystemsEZ1M.get_device_info() to throw timeout."""
+    with patch(
+        "homeassistant.components.apsystems.config_flow.APsystemsEZ1M",
+        return_value=AsyncMock(),
+    ) as mock_api:
+        mock_api.return_value.get_device_info = AsyncMock(side_effect=TimeoutError)
+        yield mock_api
```

Comment:
You can remove this one, as you can overwrite this during the test on the mock below